.travis.yml

```diff
@@ -12,6 +12,10 @@ branches:
   only:
     - master
 
+before_install:
+  - pip install --no-cache-dir -U -r requirements-test.txt
+  - ./runcheckmigration
```

Comment:
run this check after this line: https://github.com/openwisp/openwisp-radius/blob/master/.travis.yml#L26